examples/java/src/main/java/glide/examples/StandaloneExample.java

```diff
@@ -0,0 +1,125 @@
+import glide.api.logging.Logger;
+import glide.api.models.configuration.GlideClientConfiguration;
+import glide.api.models.configuration.NodeAddress;
+import glide.api.models.exceptions.ClosingException;
+import glide.api.models.exceptions.ConnectionException;
+import glide.api.models.exceptions.ExecAbortException;
+import glide.api.models.exceptions.TimeoutException;
+
+import java.util.concurrent.CancellationException;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+
+public class StandaloneExample {
+
+    /**
+     * Creates and returns a GlideClient instance. This function initializes a
+     * GlideClient with the provided node address.
+     *
+     * @return A <code>GlideClient</code> connected to the provided nodes address.
+     */
+    public static GlideClient createClient() throws CancellationException, ExecutionException, InterruptedException {
+        String host = "localhost";
+        Integer port = 6379;
+
+        // Check GlideClientConfiguration for additional options.
+        GlideClientConfiguration config =
+                GlideClientConfiguration.builder()
+                        .address(NodeAddress.builder().host(host).port(port).build())
+                        // Enable this field if the servers are configured with TLS.
+                        //.useTLS(true);
+                        .build();
+
+        GlideClient client = GlideClient.createClient(config).get();
+        return client;
+    }
+
+    /**
+     * Executes the main logic of the application, performing basic operations such as SET, GET, and
+     * PING using the provided GlideClient.
+     *
+     * @param client An instance of <code>GlideClient</code>.
+     */
+    public static void appLogic(GlideClient client) throws ExecutionException, InterruptedException {
+
+        // Send SET and GET
+        CompletableFuture<String> setResponse = client.set("foo", "bar");
+        Logger.log(Logger.Level.INFO, "app", "Set response is " + setResponse.get());
+
+        CompletableFuture<String> getResponse = client.get("foo");
+        Logger.log(Logger.Level.INFO, "app", "Get response is " + getResponse.get());
+
+        // Send PING to the primary node
+        CompletableFuture<String> pong = client.ping();
+        Logger.log(Logger.Level.INFO, "app", "Ping response is " + pong.get());
+    }
+
+    /** Executes the application logic with exception handling. */
+    private static void execAppLogic() {
+
+        while (true) {
+            GlideClient client = null;
+            try {
+                client = createClient();
+                appLogic(client);
+
+            } catch (CancellationException e) {
+                Logger.log(Logger.Level.ERROR, "glide", "Request cancelled: " + e.getMessage());
+                throw e;
+            } catch (InterruptedException e) {
+                Logger.log(Logger.Level.ERROR, "glide", "Client creation interrupted: " + e.getMessage());
```

Comment:
Can the InterruptedException only occur during client creation or can it also occur inside appLogic?

Reply:
In my current implementation, if InterruptedException occurs either in createClient() or appLogic(), this exception will be caught and logged here.

Reply:
The error message indicates that the error occurred during client creation but it could have been in appLogic instead. Can you update to  just "Client interrupted:" or "Interrupted:"?